contracts/TrueCurrencies/TrueRewardBackedToken.sol

```diff
@@ -323,16 +326,14 @@ contract TrueRewardBackedToken is RewardTokenWithReserve {
     function _setDistribution(uint256 proportion, address finOp) internal {
         require(proportion <= maxRewardProportion, "exceeds maximum proportion");
         require(_rewardDistribution[msg.sender].length == 0, "already enabled");
-        _rewardDistribution[msg.sender].push(
-            RewardAllocation(proportion, finOp));
+        _rewardDistribution[msg.sender].push(RewardAllocation(proportion, finOp));
     }
 
     /**
      * @dev Remove reward distribution for a financial opportunity
      * Remove
      */
-    function _removeDistribution(address finOp) internal {
-        delete _rewardDistribution[msg.sender][0];
-        _rewardDistribution[msg.sender].length--;
+    function _removeDistribution() internal {
```

Comment:
can we create a separate PR for this change? try to keep this relevant to only migration & mono repo.

Reply:
The compiler started to warn loudly about unused arguments in newest version. And we don't really have any other place where we are ready for changing for multiple finOps